### DISCUSSION TITLE
pass a reanimated value through props?

### DISCUSSION THREAD

**Discussion by pastean** [software-mansion/react-native-reanimated#2817] [Q&A]
I'm trying use the [position prop](https://reactnavigation.org/docs/5.x/material-top-tab-navigator/#position) material top tabs provides, which maps the current horizontal scroll position of the tab bar to a Reanimated Value, but it gives the crash explained in the post below. Initially I thought it might be an issue with the library below, but I tried passing down the animated value through props and I had similar crashes. 

The only way I got it to work was to have every single component that's supposed to use the position in the same file with the material top tabs. So maybe my issue is "how to pass a reanimated value through props".

https://github.com/nandorojo/shared-animations/issues/9

Any idea that might point me in the right direction is greatly appreciated.

**Comment by nandorojo** [comment]
Just pass it down normally, but don't read the `.value` unless it's in a reanimated hook.
